features/option-list.feature

```diff
@@ -64,3 +64,56 @@ Feature: List WordPress options
       sample_test_field_one
       """
 
+  Scenario: List option with sorting option
+    Given a WP install
+
+    When I run `wp option add sample_test_field_one sample_test_field_value_one`
+    And I run `wp option add sample_test_field_two sample_test_field_value_two`
+    And I run `wp option list --search="sample_test_field_*" --format=csv --orderby=option_id --order=asc`
```

Comment:
Please change line 68-72 like following, so it will be easier to read.

```
    Given a WP install
    And I run `wp option add sample_test_field_one sample_test_field_value_one`
    And I run `wp option add sample_test_field_two sample_test_field_value_two`

    When I run `wp option list --search="sample_test_field_*" --format=csv --orderby=option_id --order=asc`
```

Reply:
Got your point @miya0001